bootstrap/bootstrap.py

```diff
@@ -113,35 +122,64 @@ def ingest():
     data_for_output_mixin = dictor(cfg, OUTPUT_MIXIN_DATA, checknone=True)
 
     try:
-        tenant_id = get_tenant_id(tenant_id_url, headers)
-        class_id = get_class_id(create_class_url, headers, input_class_title, data_for_class)
-        input_mixin_id = get_mixin_id(create_mixin_url, headers, input_mixin_title, data_for_mixin, class_id,
+
+        tenant_id = get_tenant_id(tenant_id_url, copy.deepcopy(headers_for_ingestion))
+
+        class_id = get_class_id(create_class_url, copy.deepcopy(headers_for_ingestion), input_class_title, data_for_class)
+
+        input_mixin_id = get_mixin_id(create_mixin_url, copy.deepcopy(headers_for_ingestion), input_mixin_title, data_for_mixin, class_id,
                                       tenant_id, input_mixin_definition_title)
-        input_schema_id = get_schema_id(create_schema_url, headers, input_schema_title, class_id, input_mixin_id, data_for_schema)
-        input_dataset_id = get_dataset_id(create_dataset_url, headers, input_dataset_title, input_schema_id, data_for_dataset)
-        batch_id = get_batch_id(create_batch_url, headers, input_dataset_id, data_for_batch)
+
+        input_schema_id = get_schema_id(create_schema_url, copy.deepcopy(headers_for_ingestion), input_schema_title,
+                                        class_id, input_mixin_id, data_for_schema)
+
+        input_dataset_id = get_dataset_id(create_dataset_url, copy.deepcopy(headers_for_ingestion), input_dataset_title, input_schema_id, data_for_dataset)
+
+        batch_id = get_batch_id(create_batch_url, copy.deepcopy(headers_for_ingestion), input_dataset_id, data_for_batch)
+
         replace_tenant_id(original_file, file_with_tenant_id, tenant_id)
-        upload_file(create_batch_url, headers, file_with_tenant_id, input_dataset_id, batch_id)
-        close_batch(create_batch_url, headers, batch_id)
-        if is_output_schema_different:
-            output_mixin_id = get_mixin_id(create_mixin_url, headers, output_mixin_title, data_for_output_mixin,
+
+        upload_file(create_batch_url, copy.deepcopy(headers_for_ingestion), file_with_tenant_id, input_dataset_id, batch_id)
```

Comment:
Can we reduce column counts to fit line lengths in a page?  Normally around 80 - 100?